uber/custom_tags.py

```diff
@@ -445,10 +445,12 @@ def render(self, context):
 class single_day_prices(template.Node):
     def render(self, context):
         prices = ''
+        skiprest = ''
```

Comment:
@earl7399 I think this was your change, could you look at this and submit a PR to master for it if it still needs to be there?